api/server/middleware/middleware.go

```
@@ -45,3 +53,24 @@ func LoggerToFile() gin.HandlerFunc {
 		log.AccessLog.Infof("| %3d | %13v | %15s | %s | %s |", statusCode, latencyTime, clientIp, reqMethod, reqUri)
 	}
 }
+
+func GenerateJWT(claims jwt.Claims, jwtKey []byte) (string, error) {
+	token := jwt.NewWithClaims(jwt.SigningMethodHS256, claims)
+	tokenString, err := token.SignedString(jwtKey)
+	if err != nil {
+		return tokenString, err
+	}
+	return tokenString, nil
+}
+
+func JWTValid(token string, jwtKey []byte) (bool, *Claims) {
+	var claims Claims
+	t, _ := jwt.ParseWithClaims(token, &claims, func(t *jwt.Token) (interface{}, error) {
```

do not ignore the error